src/views/virtualmachines/details/tabs/configuration/VirtualMachineConfigurationTab.tsx

```diff
@@ -56,12 +58,13 @@ const VirtualMachineConfigurationTab: FC<NavPageComponentProps> = ({ vm }) => {
               title={<TabTitleText>{title}</TabTitleText>}
             >
               {activeTabKey === name && (
-                <Component instanceTypeVM={instanceTypeVM} vm={vm} vmi={vmi} />
+                <Component instanceTypeVM={instanceTypeVM} vm={vm || tourGuideVM} vmi={vmi} />
```

Comment:
I'm a bit concerned about using vm || tourGideVM; if we have an issue with the VM and it's undefined/null, it will load the tourGuideVM, which is hardcoded and will always be a valued machine. I think it is better to tackle this and consume boolean or some other prop, which we will be sure 100% of the guidedvm should be loaded